File: data/c#/pattern_24/code.cs
using System;
class Grammind
{
    static void Main()
    {
        int num;
        Console.Write("Enter the number of rows and columns for the square: ");
        num = Int32.Parse(Console.ReadLine());

        for (int i = 1; i <= num; i++)
        {
            for (int j = 0; j < num; j++)
            {
                Console.Write("{0,2:D }", j % 2);
            }
            Console.Write("\n");
        }
    }
}

File: data/c++/pattern_24/code.cpp
#include <iostream>
using namespace std;

int main(void)
{
    int num;
    cout << "Enter the number of rows and columns for the square: ";
    cin >> num;

    for (int i = 1; i <= num; i++)
    {
        for (int j = 0; j < num; j++)
        {
            cout << (j % 2) << " ";
        }
        cout << endl;
    }

    return 0;
}
